ui/src/app/job-details/panels/panels.component.html

```diff
@@ -1,13 +1,17 @@
 <div class="content">
   <div>
-      <p class="header">{{ job.name }}</p>
-      <p class="job-id">ID: <b>{{ job.id }}</b></p>
-      <span *ngFor="let l of labels">
-        <p class="text-info">{{ l + ': ' + job.labels[l] }}</p>
-      </span>
-      <span *ngFor="let f of displayedExtensions">
-        <p class="text-info">{{ f + ': ' + job.extensions[f] }}</p>
-      </span>
+    <p class="header">{{ job.name }}</p>
+    <button mat-button class="close"
+        (click)="handleClose()">
+      &#x274C;
```

Comment:
neat

Reply:
This actually rendered as a giant red X on my pixelbook for some reason. Went with the multiplication symbol instead which renders well on both of my chromes.